requirements/tests

```diff
@@ -12,3 +12,6 @@ ujson
 # Python 3 Only Handlers
 python-rapidjson; python_version >= '3'
 orjson; python_version >= '3' and python_version != '3.4' and platform_python_implementation != 'PyPy'
+
+# Python 2 Only
+mock; python_version < '3.3'
```

Comment:
This would of course go away with Python 3, but maybe we could do without this `mock` dependency at all?
Other tests, such as `tests/test_static.py`, we are employing [`pytest` monkey-patching facilities](https://docs.pytest.org/en/latest/monkeypatch.html), maybe that could be good enough?

Reply:
It is not a production dependency, it is for test only, so personally I don't see any issues. Of course, final decision is yours.

I don't know how exactly you are using pytest monkey patching because the tests are written using "old style" `unittest` tests. And old `unittest`-style tests doesn't support (pytest's amazing fixture injection)[https://docs.pytest.org/en/latest/fixture.html#fixtures-a-prime-example-of-dependency-injection].

Also, about `monkeypatch` vs `mock` - the later is much easier to use, IMHO. But, it is just IMHO, though.

Reply:
I agree with you that `mock` is easier/more modern, I prefer it too, but my comment was more about staying consistent with the existing tests, which are largely `pytest`-based.

Btw here we are actually using just `mock.sentinel`, and that only once if I understand it correctly. That could probably be done other way without dragging in `mock` just for that sole purpose?